src/backend/types.js

```diff
@@ -123,7 +123,7 @@ export type InteractionBackend = {|
 
 export type CommitDetailsBackend = {|
   commitIndex: number,
-  // Tuple of id, actual duration, and (computed) self duration
+  // An interleaved array: fiberID at [i], actualDuration at [i + 1], computed selfDuration at [i + 2].
```

Comment:
I hope you don't mind rephrasing the comment like this. The indexes are deliberately written as they are in code so that you can search for `[i + 1]` pattern and find them. Unfortunately `.push(one, two, three)` doesn't use the indexes.